src/components/AddForm.vue

```diff
@@ -120,7 +120,7 @@
                 >
                   <v-checkbox
                     :label="$t('label.start_torrent')"
-                    prepend-icon="mdi-play-pause"
+                    prepend-icon="mdi-play"
```

Comment:
reject

Reply:
What about this?

<img width="60" alt="image" src="https://user-images.githubusercontent.com/19761269/96710558-df3b1200-13b9-11eb-8447-e3e9a6b1d79f.png">

Reply:
I think `play-pause` pretty fit it.

Reply:
How are you pausing _something_ while starting it?

Reply:
I've pushed 50388a7. Should resolve this hopefully.